dagsen.ts

```diff
@@ -0,0 +1,39 @@
+export type Menu = {
+  dayname: string;
+  main?: string;
+  vegetarian?: string;
+  salad?: string;
+  soup?: string;
+  alacarte?: string;
+  extra?: string;
+};
+
+export async function fetchMenu(
+  day: 0 | 1 = dayNumber(new Date()),
+): Promise<Menu | null> {
+  try {
+    const res = await fetch(`http://api.teknolog.fi/taffa/sv/json/${day}`);
+    return res.json();
+  } catch (e) {
+    console.error("Failed to fetch menu from lunch API.", e);
+    return null;
+  }
+}
+
+function dayNumber(now: Date): 0 | 1 {
+  const weekday = now.getDay();
+  return 0 < weekday && weekday < 6 && now.getHours() > 16 ? 1 : 0;
```

Comment:
Show next days menu right as the restaurant closes at 15
```suggestion
  return 0 < weekday && weekday < 6 && now.getHours() > 15 ? 1 : 0;
```